packages/module/src/MessageBar/MessageBar.scss

```diff
@@ -131,3 +131,16 @@
     padding-block-end: var(--pf-t--global--spacer--md);
   }
 }
+
+// ============================================================================
+// High contrast styles
+// ============================================================================
+:root:where(.pf-v6-theme-high-contrast) {
+  .pf-chatbot__message-bar {
+    box-shadow: inset 0 0 0 1px var(--pf-t--global--border--color--default);
+
+    &:focus-within {
+      box-shadow: inset 0 0 0 2px var(--pf-t--global--color--brand--default);
```

Comment:
I'm curious about using box-shadow rather than a border. This won't make a border in forced-color mode (windows high contrast).
There are high contrast tokens for controls that adjust border width, and also special high-constrast border width tokens that are 0 in default mode, if either of those helps here.